bedrock/firefox/views.py

```diff
@@ -541,6 +542,17 @@ def get_template_names(self):
         return [template]
 
 
+def fxos(request):
```

Comment:
Well, the new page should go live sometime Sunday so but, I assume it is going to be merged before so, probably some waffle magic required.